src/renderer/qssWidget/js/qssOfficeWidget.js

```diff
@@ -0,0 +1,391 @@
+                                type: "fluid.transforms.free",
+                                func: "gpii.qssWidget.office.getRepeaterItems",
+                                args: [
+                                    "{office}.model.setting"
+                                ]
+                            }
+                        }
+                    },
+                    dynamicContainerMarkup: {
+                        container: "<div role=\"radio\" class=\"%containerClass fl-qssWidgetMenu-item fl-focusable\" tabindex=\"0\"></div>",
+                        containerClassPrefix: "flc-qssWidgetMenu-item"
+                    },
+                    handlerType: "gpii.qssWidget.office.presenter",
+                    markup: null,
+                    styles: {
+                        disabled: "disabled"
+                    },
+                    events: {
+                        onItemFocus: null
+                    },
+                    listeners: {
+                        "onCreate.enable": {
+                            this: "{that}.container",
+                            method: "removeClass",
+                            args: ["{that}.options.styles.disabled"]
+                        }
+                    }
+                }
+            },
+            closeTimer: {
+                type: "gpii.app.timer",
+                options: {
+                    listeners: {
+                        "onTimerFinished.closeWidget": {
+                            func: "{qssWidget}.close",
+                            args: ["{office}.keyboardEvent"]
+                        }
+                    }
+                }
+            }
+        },
+        listeners: {
+            "onCreate.visibilityChange": {
+                funcName: "gpii.qssWidget.office.addVisibilityChangeListener",
+                args: ["{closeTimer}"]
+            },
+            "onDestroy.visibilitychange": {
+                funcName: "gpii.qssWidget.office.removeVisibilityChangeListener"
+            }
+        },
+        invokers: {
+            calculateHeight: {
+                funcName: "gpii.qssWidget.calculateHeight",
+                args: [
+                    "{qssWidget}.container",
+                    "{that}.dom.menuControlsWrapper",
+                    "{that}.dom.heightListenerContainer"
+                ]
+            },
+            close: {
+                funcName: "gpii.qssWidget.office.close",
+                args: [
+                    "{that}",
+                    "{closeTimer}",
+                    "{arguments}.0" // keyboardEvent
+                ]
+            }
+        },
+        events: {
+            onHeightChanged: null
+        }
+    });
+
+
+    /**
+     * Checks the current states and generates the appropriate command
+     * @param {Object} states - simple true/false object with the current states
+     * @param {Object} availableCommands - a simple list of available commands defined in the model
```

Comment:
Not a list - it actually looks like a hash of strings to string? This can be Object.<string,string>
https://jsdoc.app/tags-type.html

Reply:
Done, changed everywhere.